openmc/mesh.py

```diff
@@ -1937,3 +1938,25 @@ def from_xml_element(cls, elem):
         length_multiplier = float(get_text(elem, 'length_multiplier', 1.0))
 
         return cls(filename, library, mesh_id, '', length_multiplier)
+
+
+def read_meshes(elem):
```

Comment:
Since this is only used in settings.py, why not just move it there? Also, since it is really not intended to used by a user, please change the name to `_read_meshes` if it's going to live at module scope.

Reply:
I'm thinking that there are other places we might be able to use this as well going forward and because it pertains only to meshes I think it makes sense to keep it in this module. That'd be my vote anyways. Feel free to override 😄 

I do see this as a utility function for I/O operations within our classes, so I'll go ahead and make that name change. Thanks for catching that!